### ISSUE TITLE
histogram

### ISSUE THREAD

**Issue by daavoo** [iterative/dvc#6611]

**Comment by mnrozhkov** [comment]
@daavoo could you describe your use case? Is it critical for you to plot few revisions on the same plot? or it doesn't make practical sense?

**Comment by daavoo** [comment]
> @daavoo could you describe your use case? 

Didn't have a particular use case in mind but just wanted to bring this into the current discussion about future of plots  (https://github.com/iterative/dvc/discussions/5980) . Nevertheless, I see al least two common use cases (one high level, one low level):

- Distribution of scores 
For example, store an array of model score predictions on a validation set in order to choose a threshold value.

- Monitoring model training
For example, store the gradients/weights of some layer(s) of a neural network at each epoch to debug training, like checking how different initialization strategies affect.

> Is it critical for you to plot few revisions on the same plot? or it doesn't make practical sense?

I would say that not only revisions but having the ability to show multiple histograms in the same plot in general would make sense for both use cases (i.e. same revision but one histogram for each class in the "distribution of scores" case)
